### ISSUE TITLE
 Accessibility in cv-dropdown-item

### ISSUE THREAD

**Issue by barbarabeat** [carbon-design-system/carbon-components-vue#1343]
## Detailed description

> Describe in detail the issue you're having.

Run IBM Equal Access Accessibility Checker on the cv-dropdown-item, result Violation: “”The element with role "menuitemradio" is not contained in or owned by an element with one of the following roles: "group, menu, menubar””

Cv-dropdown-item don not inserts a role in the ‘li’ element.

> Is this a feature request (new component, new icon), a bug, or a general issue?

accessibility issue

> Is this issue related to a specific component?

cv-dropdown-item

> What did you expect to happen? What happened instead? What would you like to see changed?

Cv-dropdown-item creates an role=“menuitem” in the “li”. 

> What browser are you working in?

firefox

> What version of the Carbon Design System are you using?

latest

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?
--

## Steps to reproduce the issue

1. use Cv-dropdown-item component
2. Run IBM Equal Access Accessibility Checker
3. Open violations and go to 4.1.2 Name, Role, Value
<img width="607" alt="Captura de Tela 2022-07-18 às 12 54 40" src="https://user-images.githubusercontent.com/25625728/179553191-e73e9891-b1c5-4f0e-bca3-ea357fcf881e.png">

4. etc.

## Additional information

- Screenshots or code
- Notes
